meta-genivi-dev/recipes-sota/rvi/rvi_git.bbappend

```diff
@@ -0,0 +1,22 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI += "\
+    file://rvi.service \
+    file://device_id \
+    "
+
+do_install_append () {
+    install -d "${D}${systemd_unitdir}/system"
+    install -m 0644 "${WORKDIR}/rvi.service" "${D}${systemd_unitdir}/system/"
+    install -m 0644 "${WORKDIR}/device_id" "${D}${sysconfdir}/opt/rvi/device_id"
+    rm ${D}${sysconfdir}/init.d/rvi
+}
+
+inherit systemd
+
+INITSCRIPT_NAME = ""
+INITSCRIPT_PARAMS = ""
+
+pkg_postinst_${PN} () {
```

Comment:
The rvi package generates a random uuid after it is installed. This function overwrites that so instead nothing happens. This way you can add your device id to `recipes-sota/rvi/files/device_id` and it won't get blitzed.